### ISSUE TITLE
Serialize cell contents

### ISSUE THREAD

**Issue by juanfcocontreras** [man-group/dtale#466]
dtale is currently not able to show pandas DataFrames containing lists or dictionaries in any cell.

Would it be possible to serialize the content of those elements to be able to display those DataFrames?

A solution could be to convert their content to String. Another less practical solution would be to display those cells as "Object" or similar, but at least you can work with the rest of the DataFrame in dtale.

Thank you very much!

Minimal sample:

```
import pandas as pd
from dtale import show

data = [[1, 2, 3],]
s = pd.Series(data)
df = pd.DataFrame(s)

show(df)
```

```
pandas/core/algorithms.py in unique(values)
    405
    406     table = htable(len(values))
--> 407     uniques = table.unique(values)
    408     uniques = _reconstruct_data(uniques, original.dtype, original)
    409     return uniques

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable.unique()

pandas/_libs/hashtable_class_helper.pxi in pandas._libs.hashtable.PyObjectHashTable._unique()

TypeError: unhashable type: 'list'
```

**Comment by aschonfeld** [comment]
Oooh I like this one!  Yea I’ll work on seeing if I can update the JSON serializer to handle this.

**Comment by aschonfeld** [comment]
Boom
![image](https://user-images.githubusercontent.com/11547371/115771883-776f7f00-a37c-11eb-80f7-f039c2354105.png)

**Comment by aschonfeld** [comment]
Just released 1.44.0 with this fix

**Comment by juanfcocontreras** [comment]
Thank you so much!
